lib/internal/assert/assertion_error.js

```diff
@@ -96,47 +96,47 @@ function createErrDiff(actual, expected, operator) {
        (typeof actual === 'function' && typeof expected === 'function'))) {
     operator = 'strictEqualObject';
   }
-
```

Comment:
why are these blank lines removed, here and below?

Reply:
Ohh I think I had put a few lines of code in those spaces for debugging, must have removed the blank lines along with it